<?xml version="1.0" ?>
<launch>
  <arg name="robot_ip" default="panda"/>
  <arg name="load_gripper" default="true" />
  <arg name="sim" default="false" />
  <arg name="controller" default="cartesian_velocity_controller">

  <include file="$(find panda_interface)/launch/franka_control.launch" >
    <arg name="robot_ip" value="$(arg robot_ip)" />
    <arg name="load_gripper" value="$(arg load_gripper)" />
  </include>

  <rosparam command="load" file="$(find panda_ros_controllers)/config/franka_controllers.yaml" />
  <rosparam  ns="cartesian_pose_impedance_controller" command="load" file="$(find panda_ros_controllers)/config/cartesian_pose_impedance_additional_params.yaml"/>

  <node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false" output="screen"
        args="position_joint_trajectory_controller"/>
  <node name="controller_spawner2" pkg="controller_manager" type="spawner" respawn="false" output="screen"
      args="--stopped $(arg controller)"/>

</launch>

